draft-ietf-mimi-content.md

```diff
@@ -1279,6 +1329,87 @@ Person & email address to contact for further information:
+purpose for which it was defined. Careful reading of the documentation for the
+mechanism is necessary to understand the applicability of that mechanism. The
+IETF could recommend mechanisms that have limited applicability, but will
+provide applicability statements that describe any limitations of the mechanism
+or necessary constraints on its use.
+    - N: Indicates that the item has not been evaluated by the IETF and that the
+IETF has made no statement about the suitability of the associated mechanism.
+This does not necessarily mean that the mechanism is flawed, only that no
+consensus exists. The IETF might have consensus to leave an item marked as "N"
+on the basis of it having limited applicability or usage constraints.
+    - D: Indicates that the item is discouraged and SHOULD NOT or MUST NOT be
+used. This marking could be used to identify mechanisms that might result in
+problems if they are used, such as a weak cryptographic algorithm or a mechanism
+that might cause interoperability problems in deployment.
+- Reference: The document where this MIMI content extension is defined
+
+Initial Contents:
+
+| Key   | Name                                     | Type      | R | Reference |
+|:------|:-----------------------------------------|:----------|:--|:----------|
+| 0     | (reserved)                               | -         | - | RFCXXXX   |
+
+### Expert Review
+
+Expert Review [@!RFC8126] registry requests are registered
+after a three-week review period on the MIMI Designated Expert (DE) mailing list
+<mimi-reg-review@ietf.org> on the advice of one or more of the MIMI DEs.
+
+Registration requests sent to the MIMI DEs' mailing list for review
+SHOULD use an appropriate subject (e.g., "Request to register value
+in MIMI Content Extensions Keys registry").
+
+Within the review period, the MIMI DEs will either approve or deny
+the registration request, communicating this decision to the MIMI DEs'
+mailing list and IANA. Denials SHOULD include an explanation and, if
+applicable, suggestions as to how to make the request successful.
+Registration requests that are undetermined for a period longer than
+21 days can be brought to the IESG's attention for resolution using
+the <iesg@ietf.org> mailing list.
+
+Criteria that SHOULD be applied by the MIMI DEs includes determining
+whether the proposed registration duplicates existing functionality,
+whether it is likely to be of general applicability or useful only
+for a single application, and whether the registration description
+is clear.
+
+IANA MUST only accept registry updates from the MIMI DEs and SHOULD
+direct all requests for registration to the MIMI DEs' mailing list.
+
+In cases where a registration decision could be perceived as creating a conflict
+of interest for a particular MIMI DE, that MIMI DE SHOULD defer to the judgment
+of the other MIMI DEs.
+
+
 # Security Considerations
 
 ## General handling
```

Comment:
I think the list below still needs to be updated:

  > - a topic is specified, but an `inReplyTo` or `replaces` field refers to a
    message outside of the topic
    
This can be removed?
    
> * expires
>   - refers to a date more than a year in the past
>   - refers to a date more than a year in the future

Should this differentiate between relative and absolute timestamps?

Reply:
> I think the list below still needs to be updated:
> 
> > * a topic is specified, but an `inReplyTo` or `replaces` field refers to a
> >   message outside of the topic
> 
> This can be removed?

yes

> > * expires
> >   
> >   * refers to a date more than a year in the past
> >   * refers to a date more than a year in the future
> 
> Should this differentiate between relative and absolute timestamps?

I can add an extra sentence or two. More than a year in the future is possible with both absolute and relative timestamps. More than a year in the past is only possible with absolute timestamps.

Reply:
Fixed.